docs/cstor.md

```diff
@@ -0,0 +1,384 @@
+
+<br>
+
+### cStor pool spec details
+
+A cStor pool spec consists of
+
+- Number of pools
+- List of nodes that host the pools
+- List of disks on each node that constitute the pool on that given node
+- RAID type within the pool (currently stripe and mirror are supported). Refer to the [cStor Pool roadmap](/docs/next/cStorPools.html#cstor-pool-feature-roadmap) to find the status of RAIDz1 support
+
+**Number of pools:** It is good to start with 3 poos as the number of volume replicas will be typically three or one. However, the number of pools are fixed in OpenEBS 0.8 version. Support for increasing the pool replicas on the fly is in the [cStor Pool roadmap](/docs/next/cStorPools.html#cstor-pool-feature-roadmap)  . At the time of cStor pool creation, individual and independent pools are created on the specified nodes. 
+
+**List of nodes that host the pools:** This information and the number of pool replicas are implicitly provided by analyzing the provided disk CRs in the spec file. For example, if the spec file has 3 disk CRs, which belong to 3 different nodes, it implicitly means the number of pool replicas are 3 and the list of nodes taken from the disk CR metadata.
+
+**List of disks:** This is perhaps the most important information in the pool specification. The disk CRs need to be listed and carefully taken by first determining the list of nodes and the number of disks on each node. 
+
+**Type of pool:** This is also another important configuration information in the pool specification. It defines how the disks are pooled together within a node. Possible configurations are 
+
+- STRIPE
+- MIRROR
+- RAIDZ1 (Roadmap)
+- RAIDZ2 (Roadmap)
+
+***Note:*** A pool cannot be extended beyond a node. When the pool type is STRIPE, it should not be assumed that data is striped across the nodes. The data is striped across the disks within a the pool on that given node. As mentioned in the data replication section, the data is synchronously replicated to as many number of pools as the number of volume replicas irrespective of the type of pool.  
+
+<br>
+
+### Operations on cStor Pool 
+
+cStor Pool is an important component in the storage management. It is fundamental to storage planning especially in the context of hyperconvergence planning. The following are the different operations that can be performed on cStor Pools.
+
+
+
+**Create a Pool :** Create a new pool with all the configuration. It is typical to start a pool with 3 pool replicas on three nodes. Currently pool types supported are STRIPE and MIRROR. A pool needs to be created before storage classes are created. So, pool creation is the first configuration step in the OpenEBS configuration process.
+
+**Add a new pool replica** : (Not supported in OpenEBS 0.8, refer to the  [cStor roadmap](/docs/next/cstor.html#cstor-roadmap)  ) .A new pool replica may need to be added for many different reasons. Some of them could be 
```

Comment:
To avoid the confusion between Pool and -- pool replica, can the `pool replica` be replaced with `replica store` or `replica instance`?